paddle/pybind/pybind.cc

```diff
@@ -40,6 +42,23 @@ static size_t UniqueIntegerGenerator() {
   return generator.fetch_add(1);
 }
 
+std::once_flag gflags_init_flag;
+
+void InitGflags(std::vector<std::string> argv_vec) {
```

Comment:
I think we can add InitGFlags into a new .cc files so could we prevent from making this pybind.cc infinitely long. @reyoung had an idea how to separate pybind.cc into multiple files.

Reply:
Yes, we actually have some initialization work to do before executing the neural network, such as glog/gflags settings, and device count/information detection. I will add a TODO in this code and move it to `init.cc` later.